othello/apps/tournaments/utils.py

```diff
@@ -8,10 +8,11 @@ def chunks(v, n):
 
 def make_pairings(players, bye_player):
     matches = []
-    players = players.order_by("-ranking")
+    player_count = players.count()
```

Comment:
This is pointless. You're about to evaluate the set; just `len()` it then. Doing this wastes a query.